pkg/networkmanager/firewall.jsx

```diff
@@ -184,6 +184,7 @@ function ZoneSection(props) {
         {props.zone.services.length > 0 &&
         <CardBody className="contains-list">
             <ListingTable columns={[{ title: _("Service"), props: { width: 40 } }, { title: _("TCP"), props: { width: 30 } }, { title: _("UDP"), props: { width: 30 } }]}
+                          id={props.zone.id}
```

Comment:
This looks curious at first sight, as `id=` is already a standard HTML attribute. But it does end up as actual `id` of the generated `<table>`, so seems ok.